geomesa-filter/src/test/scala/org/locationtech/geomesa/filter/OrSplittingFilterTest.scala

```diff
@@ -26,6 +26,20 @@ class OrSplittingFilterTest extends Specification {
 
   def splitFilter(f: Filter) = osf.visit(f, null)
 
+  implicit class RichFilter(val filter: Filter) {
+    def &&(that: Filter) = ff.and(filter, that)
+
+    def ||(that: Filter) = ff.or(filter, that)
+
+    def ! = ff.not(filter)
+  }
+
+  implicit def stringToFilter(s: String): Filter = ECQL.toFilter(s)
```

Comment:
Is this still used?  Or did all the implicit uses become explicit?

Reply:
i copied it into the test instead of making everything explicit - it was pretty heavily used in this particular test